ktor-server/ktor-server-host-common/jvmAndNix/src/io/ktor/server/engine/ApplicationEngine.kt

```diff
@@ -37,6 +37,16 @@ public interface ApplicationEngine {
          * Specifies the minimum size of a thread pool used to process application calls.
          */
         public var callGroupSize: Int = parallelism
+
+        /**
+         * Specifies the maximum amount of time for activity to cool down in milliseconds.
+         */
+        public var gracePeriodMillis: Long = 500
```

Comment:
I don't see any usages of it. Why make it public property?

Reply:
It is used in `public fun BaseApplicationEngine.Configuration.loadCommonConfiguration`

Reply:
But it's only set there and not read anywhere

Reply:
But why is it bad that it is public?

Reply:
I don't understand why it should be. There is nothing users can do with it, and we don't use it as well, only set a value to it.

Reply:
Understand, I agree that it is unnecessary